.github/workflows/args.yaml

```diff
@@ -4,13 +4,23 @@ on:
   # Run CI on pushes to the main branch, and on PRs against main.
   push:
     branches: [ main ]
+    paths:
+      - '.github/workflows/args.yaml'
+      - 'pkgs/args/**'
   pull_request:
     branches: [ main ]
+    paths:
+      - '.github/workflows/args.yaml'
+      - 'pkgs/args/**'
   schedule:
     - cron: "0 0 * * 0"
 env:
   PUB_ENVIRONMENT: bot.github
 
+defaults:
```

Comment:
Ah, this is nice; I've been setting the cwd for each job config. We should definitely prefer to do as a workflow file default.

Reply:
Same, just tried this and it seems to work.